cin_validator/rules/cin2022_23/rule_8869.py

```diff
@@ -50,11 +50,14 @@ def validate(
     # If <AssessmentFactors> (N00181) = “21”, this must be the only <AssessmentFactors> (N00181) present.
     df_orig = df.copy()
 
-    df = df[df[AssessmentFactors] == "21"]
+    df = df[(df[AssessmentFactors] == "21") | (df[AssessmentFactors] == 21)]
 
     #  Merge tables
     df = df.merge(
-        df_orig, how="left", on=["LAchildID", "CINdetailsID"], suffixes=["", "_orig"]
+        df_orig,
```

Comment:
This assumes that no two cin episodes of  a child can ever have the same authorization date. Which should be true in real life but might happen in the data.
I say it doesn't hurt to leave the check in.